com.zenorogue.HyperRogue.json

```diff
@@ -0,0 +1,58 @@
+{
+  "app-id":"com.zenorogue.HyperRogue",
+  "runtime":"org.freedesktop.Platform",
+  "runtime-version":"20.08",
+  "sdk":"org.freedesktop.Sdk",
+  "command":"hyperrogue",
+  "finish-args":[
+    "--device=all",
```

Comment:
may I assume this is for game controllers?

Reply:
Yes